src/helpers/stringUtil.js

```diff
@@ -46,18 +46,13 @@ export function getTxTypeString(txType, intl) {
   const { formatMessage } = getIntlProvider(intl.locale, intl.messages);
 
   switch (txType) {
-    case TransactionType.APPROVE_CREATE_EVENT:
-    case TransactionType.APPROVE_SET_RESULT:
-    case TransactionType.APPROVE_VOTE: {
-      return formatMessage(strings.approveBotTransfer);
```

Comment:
please remove the strings if unused anywhere else. same goes for the other strings below.